internal/flagutils/cli-flags.go

```diff
@@ -0,0 +1,15 @@
+package flagutils
```

Comment:
I would rather have this as part of the commands themselves than have a entire package just for this.

Reply:
I was trying to remove duplicate code, but I can remove the package. I'm OK with either approach

Reply:
> I was trying to remove duplicate code, but I can remove the package. I'm OK with either approach

The code below will be in 3 different places
```go
&cli.BoolFlag{
		Name:  "skip-host-preflights",
		Usage: "Skip host preflight checks. This is not recommended unless you are sure your system is compatible.",
		Value: false,
	}
```